Navaids/FIXES_UK.txt

```diff
@@ -5,7 +5,7 @@ ABEVI N054.42.27.000 W002.46.29.240
 ABGEP N053.25.39.320 W004.51.39.710
 ABKAT N054.08.53.120 W001.58.46.690
 ABKIM N050.50.43.000 W001.58.58.000
-ABLIN N052.46.57.860 W004.59.33.380
+ABLIN N052.46.58.000 W004.59.33.000
```

Comment:
https://www.aurora.nats.co.uk/htmlAIP/Publications/2023-08-10-AIRAC/html/eAIP/EG-ENR-4.4-en-GB.html#ENR-4.4 has different, more precise values for these. I haven't checked the rest, but like I say I don't expect them to be changing as I think they were all accurate.

Reply:
Oh yes it'll be just the french ones I copied back over, I'll fix up those that I changed.